File: P2N_Pet_BackEnd/P2N_Pet_API/P2N_Pet_API/Database/PetShopModels/Order.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace P2N_Pet_API.Database.PetShopModels
{
    public partial class Order
    {
        public Order()
        {
            Cartitems = new HashSet<Cartitem>();
        }

        public ulong Id { get; set; }
        public ulong? Cartid { get; set; }
        public ulong? Customerid { get; set; }
        public int? Statusorderid { get; set; }
        public ulong? Totalmoney { get; set; }
        public string Note { get; set; }
        public int? Status { get; set; }
        public ulong? Createuser { get; set; }
        public DateTime? Createdate { get; set; }
        public ulong? Updateuser { get; set; }
        public DateTime? Updatedate { get; set; }

        public virtual Cart Cart { get; set; }
        public virtual Customer Customer { get; set; }
        public virtual Statusorder Statusorder { get; set; }
        public virtual ICollection<Cartitem> Cartitems { get; set; }
    }
}

File: P2N_Pet_FrontEnd/P2N-Pet/projects/admin/src/app/models/order.ts
export class Order{
    public Id: number;
    public CustomerName: string;
    public CustomerPhone: string;
    public CustomerEmail: string;
    public TotalMoney: number;
    public Note: string;
    public StatusText: string;
    public CreateUserName: string;
    public CreateDate: string;
    public UpdateUserName: string;
    public UpdateDate: string;
}

export class OrderCondition {
    CustomerName: string;
    CustomerPhone: string;
    CustomerEmail: string;
    StatusOrderId: string;
    Status: string;
  
    constructor() {
      this.CustomerName = '';
      this.CustomerPhone = '';
      this.CustomerEmail = '';
      this.StatusOrderId = '1';
      this.Status = '0';
    }
  }